libpermian/plugins/run_subset/test.py

```diff
@@ -24,6 +24,7 @@ def test_no_args(self):
                     'testcases_queries': None,
                     'configurations': None,
                     'crc_queries': None,
+                    'display_name': None
```

Comment:
Please add ',' at the end of the line so that this line doesn't need to be modified if there's anything else added later.